build_projects/shared-build-targets-utils/Utils/HostVersion.cs

```diff
@@ -72,7 +72,7 @@ public override string ToString()
         // These versions should only be incremented in a servicing release if the package in question
         // is being updated.
         public VerInfo LatestHostVersion => new VerInfo(1, 0, 1, "", "", "", CommitCountString);
-        public VerInfo LatestHostFxrVersion => new VerInfo(1, 0, 1, "", "", "", CommitCountString);
+        public VerInfo LatestHostFxrVersion => new VerInfo(1, 0, 5, "", "", "", CommitCountString);
```

Comment:
The point I was making is that we do not change these values unless we are updating the nuget package and that happens if hostfxr.dll is changing.

In this case, we are not fixing any issue in hostfxr.dll, so we dont have to change the version here. Instead we should change the value passed to https://github.com/dotnet/core-setup/pull/1838/files#diff-6a92b73d69f92c6c99098e4630dcb696R142.